src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -250,11 +253,27 @@ private TranscriptionWorkflowEntity saveTranscriptionWorkflow(UserAccountEntity
         transcriptionWorkflow.setTranscriptionStatus(transcriptionStatus);
         transcriptionWorkflow.setWorkflowActor(userAccount);
         transcriptionWorkflow.setWorkflowTimestamp(OffsetDateTime.now(UTC));
-        transcriptionWorkflow.setWorkflowComment(workflowComment);
+
+
         transcriptionWorkflow.setCreatedBy(userAccount);
         transcriptionWorkflow.setLastModifiedBy(userAccount);
 
-        return transcriptionWorkflowRepository.saveAndFlush(transcriptionWorkflow);
+        TranscriptionWorkflowEntity savedTranscriptionWorkFlow = transcriptionWorkflowRepository.saveAndFlush(
+            transcriptionWorkflow);
+
+        if (!StringUtils.isBlank(workflowComment) && !StringUtils.isEmpty(workflowComment)) {
```

Comment:
Don't think we need the isEmpty condition check as well since isBlank does
`Checks if a CharSequence is empty (""), null or whitespace only`?

Reply:
ACCEPT